api/src/middleware/publications.js

```diff
@@ -103,19 +103,18 @@ async function validateAuthorId(req, res, next) {
       ]),
     );
   }
-  await axios
-    .get(`https://scholar.google.com.sg/citations?user=${_id}`)
-    .then(() => { next(); })
-    .catch((error) => {
-      // if you mess around with the user id you only get 404
-      next(
-        fillErrorObject(
-          error.response.status,
-          'Validation error',
-          ['No Google Scholar user profile found with the given id'],
-        ),
-      );
-    });
+
+  try {
+    await axios(`${playwrightConfig.baseUrl}${_id}`);
```

Comment:
Making this read `axios.get()` should be more readable.
So the error was because we weren't using `playwrightConfig.baseUrl`?

Reply:
Will do. I think it could've been part of it since it was using .com.sg but it still doesn't make sense why it would start returning 403 now and not before. Hoping this will fix it in prod.